chromium_src/chrome/browser/ui/views/tabs/tab_strip.cc

```diff
@@ -4,10 +4,21 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "chrome/browser/ui/views/tabs/tab_strip.h"
+
+#include "brave/browser/ui/views/tabs/brave_tab.h"
 #include "brave/browser/ui/views/tabs/brave_tab_hover_card_controller.h"
+#include "chrome/browser/ui/views/tabs/browser_tab_strip_controller.h"
+#include "chrome/browser/ui/views/tabs/tab_container.h"
+
+#if BUILDFLAG(IS_WIN)
+#include "ui/gfx/win/hwnd_util.h"
```

Comment:
why this include?

Reply:
Without it the build was failing on Windows. Somewhere deep inside here there's a Windows `AddTab` function which was getting replaced by our overriding definition of `AddTab`. This fixes that by including the file first.